packages/docutils/lib/nav.ts

```diff
@@ -0,0 +1,165 @@
+import _ from 'lodash';
+import path from 'node:path';
+import {fs} from '@appium/support';
+import {
+  findMkDocsYml,
+  findTypeDocJson,
+  readTypedocJson,
+  readYaml,
+  relative,
+  safeWriteFile,
+  stringifyYaml,
+} from './util';
+import {DEFAULT_REL_TYPEDOC_OUT_PATH} from './constants';
+import logger from './logger';
+import {MkDocsYml} from './model';
+import {IterableElement} from 'type-fest';
+
+const DEFAULT_REFERENCE_HEADER = 'Reference';
+
+const log = logger.withTag('mkdocs-nav');
+
+/**
+ *
+ * @param opts - Options
+ * @todo implement `dryRun` option
+ */
+export async function updateNav<S extends string>({
```

Comment:
for your sanity you may want to skip this function

Reply:
but what it ends up doing is it looks for anything in the "Reference" header (or whatever you want to call it) and finds stuff in the `docs/reference/commands` folder, and rewrites anything with those values, leaving the other stuff in place. it also supports _not_ using a header, where nav is just an array of strings